### ISSUE TITLE
ChatSecure is crashed

### ISSUE THREAD

**Issue by ivaneliseeff** [ChatSecure/ChatSecure-iOS#965]
Hello!

The application crashes after trying to download the transmitted image.
Tested on 4.2.1 (109) version of app.

Devices: iPhone 7, iOS 11.2.6 and iPhone X iOS 11.2.6

**Comment by ivaneliseeff** [comment]
Images are not saved in the camera roll. transfer to other applications, no problem

**Comment by mimi89999** [comment]
Hello. Do you still encounter this issue in the latest ChatSecure version?

**Comment by ivaneliseeff** [comment]
> 
> 
> Hello. Do you still encounter this issue in the latest ChatSecure version?

I have been using another solution for a long time. it is safer and more stable.

**Comment by licaon-kter** [comment]
@ivaneliseeff it is a secret?

**Comment by ivaneliseeff** [comment]
> 
> 
> @ivaneliseeff it is a secret?

no. rocketchat. for my task - the best solution

**Comment by kruthoff** [comment]
> Hello. Do you still encounter this issue in the latest ChatSecure version?

Yes. The latest version 4.3.9 9143) still has this issue. It is not possible to save a picture to the iPhone. The app crashes.
